tox.ini

```diff
@@ -29,6 +29,7 @@ deps =
     ase==3.22.1
 
 commands =
+    pip install chemiscope[explore]
```

Comment:
ideally we should not use pip in the commands section of tox. If you need to use the extra dependencies, you should be able to use `extras = explore`: https://tox.wiki/en/latest/config.html#extras